backend/src/main/java/com/benine/backend/camera/ipcameracontrol/IPCamera2.java

```diff
@@ -0,0 +1,175 @@
+package com.benine.backend.camera.ipcameracontrol;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.json.simple.JSONObject;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import com.benine.backend.camera.Camera;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.MovingCamera;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ZoomingCamera;
+
+public class IPCamera2 implements Camera, MovingCamera, ZoomingCamera {
+  
+  String ipaddress;
+
+  private int id = -1;
+
+  /**
+   *  Create a new IP Camera object.
+   *  @param ip address of this camera.
+   */
+  public IPCamera2(String ip) {
+    ipaddress = ip;
+  }
+
+  @Override
+  public String sendCommand(String command) {
+    try {
+      InputStream com = new URL("http://" + ipaddress + "/cgi-bin/CGIProxy.fcgi?cmd=" + command + "&usr=user&pwd=geheim").openStream();
+      DocumentBuilderFactory dbFactory  = DocumentBuilderFactory.newInstance();
+      DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+      Document doc = dBuilder.parse(com);
+      doc.getDocumentElement().normalize();
+      NodeList l = doc.getElementsByTagName("result");
+      if (Integer.parseUnsignedInt(l.item(0).getTextContent()) != 0) {
+        throw new IOException();
+      }
+      return doc.toString();
+    } catch (IOException e) {
+      e.printStackTrace();
+    } catch (SAXException e){
+      e.printStackTrace();
+    } catch (ParserConfigurationException e){
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  @Override
+  public int getZoomPosition() throws CameraConnectionException {
+    //Not supported
+    return 50;
```

Comment:
Wouldn't it be better to return a value like -1 if an action is not supported? So the UI always takes a default value on -1?